.circleci/config.yml

```diff
@@ -2,12 +2,13 @@ version: 2
 jobs:
   test:
     docker:
-      - image: nathanielc/flux-build:latest
+      - image: nathanielc/flux-build:rust
```

Comment:
Is this your image id?

Did you push a new one built from `Dockerfile_build` with that tag?

Reply:
The image is under my account since we don't have a company account that I know of. The image is auto built from the `flux` repo, but inorder to CI to pass before the changes are merged to master I built  the image locally and pushed it with the `rust` tag. Once this all merges to master we can  edit that line again to say `latest`.